packages/docusaurus/src/client/baseUrlIssueBanner/styles.module.css

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.baseUrlIssueBanner {
+  display: none;
```

Comment:
Curious as to why this was hidden from view? Getting this error and it never showed which caused me to come here :/